packages/charrua-core/charrua-core.0.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mirage/charrua-core/releases/download/v0.5/charrua-core-0.5.tbz"
```

Comment:
ohh so that is where the file come from.

I agree that is better, but then we should do it retroactively for every other release. The inconsistency on something simple like the release file we should not have.

Also, can we generate tarball ? bzip is annoying and I never have it installed anywhere.